index.js

```diff
@@ -32,8 +32,8 @@ var weekdayRange = require('./weekdayRange');
 
 module.exports = generate;
 
-// cache the `facebook/regenerator` wrapGenerator function.
-var wg = vm.runInNewContext(regenerator('', { includeRuntime: true }) + ';wrapGenerator');
+// cache the `facebook/regenerator` wrap function for the Generator object.
+var wg = vm.runInNewContext(regenerator.compile('', { includeRuntime: true }).code + ';regeneratorRuntime.wrap()');
```

Comment:
Don't invoke the function! We want a reference to the function itself, not its return value!